### ISSUE TITLE
[Question] Linux support

### ISSUE THREAD

**Issue by dzoech** [SoCreate/service-fabric-data-protection#1]
Hello, thanks for your work! I'd like to ask if this service can also be run on a Linux cluster.
Thank you!

**Comment by lurock** [comment]
I have not tested in on a Linux cluster but I don't know of any reasons why you couldn't.  If you test it out on your Linux cluster let us know if you are having any issues and maybe we can help.

**Comment by dzoech** [comment]
Thanks for your response. I'm currently evaluating different approaches to use DataProtection in SF.
Do you see any possible problems for clients running in a container?

**Comment by lurock** [comment]
The client should not have an issue running in a container.

**Comment by dzoech** [comment]
I did now try to use your library but after some research it seems (at least for me) that the client is not able to run inside a linux or windows-nanoserver container.

After installing `SoCreate.AspNetCore.DataProtection.ServiceFabric` the following exception occurred after running the container:

Removed most of the stack trace for readability (full stack trace here: https://pastebin.com/d8R1hQ6A):
``An error occurred during the activation of a particular registration. See the inner exception for details. Registration: Activator = DistributedCacheStoreLocator (ReflectionActivator), Services = [SoCreate.Extensions.Caching.ServiceFabric.IDistributedCacheStoreLocator], Lifetime = Autofac.Core.Lifetime.RootScopeLifetime, Sharing = Shared, Ownership = OwnedByLifetimeScope ---> An exception was thrown while invoking the constructor 'Void .ctor(Microsoft.Extensions.Options.IOptions`1[SoCreate.Extensions.Caching.ServiceFabric.ServiceFabricCacheOptions])' on type 'DistributedCacheStoreLocator'. ---> Unable to load DLL 'FabricClient.dll': The specified module could not be found. (Exception from HRESULT: 0x8007007E) (See inner exception for details.) ``

If I understand correctly, your usage of `Microsoft.ServiceFabric.Services.Remoting` requires the client to have the Service Fabric SDK installed.
